patches/747-mikrotik-extra-support.patch

```diff
@@ -534,6 +544,17 @@
 +       phy-mode = "rgmii";
 +};
 --- /dev/null
++++ b/target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-discg-5acd.dts
+@@ -0,0 +1,8 @@
++// SPDX-License-Identifier: GPL-2.0-only
++
++#include "qcom-ipq4018-discg-5acd.dts"
```

Comment:
Looks like this file is trying to include itself.

Reply:
Fixed in one of the later commits. Didn't see this until after the later commits.